guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -26,7 +26,7 @@ endif::[]
 ifdef::katello,satellite,orcharhino[]
 |/var/lib/pulp |1 MB |300 GB
 
-|/var/lib/containers if using {advisorengine} |10 GB |15 GB
+|/var/lib/containers if using {insights-iop} |10 GB |15 GB
```

Comment:
AFAIK, this should be `ifdef:satellite[]` only. We don't include IoP in upstream yet.